hooks/pre/10-reset_data.rb

```diff
@@ -31,14 +31,14 @@ def load_foreman_config
   db_config
 end
 
-def reset_database
+def reset_foreman_database
   Kafo::KafoConfigure.logger.info 'Dropping database!'
```

Comment:
I made a comment about the logger before, but that was meant for all instances in the entire file